simulator/func_sim/alu.h

```diff
@@ -285,6 +286,14 @@ struct ALU
         instr->v_dst[0] = ( (instr->v_src[0] >> pack_width) | (instr->v_src[1] & (bitmask<T>(pack_width) << pack_width)));
     }
 
+    template<typename T> static
+    void bclri( Instr* instr )
+    {
+        size_t index = shamt_imm( instr) & (XLEN - 1);
+        size_t mask = ~(static_cast<size_t>( 1) << index);
```

Comment:
`size_t` won’t cast to `uint128`. How did you compile it?

Reply:
Compilation successed and all tests were passed. Are there any problems with 128 bit registers?
Specification contains only RV32 and RV64 encodings for this instruction (https://github.com/riscv/riscv-bitmanip/releases/download/1.0.0/bitmanip-1.0.0-38-g865e7a7.pdf). Should I add 128 bit registers support?

Reply:
On CI, compilation failed on Linux. What is your OS, compiler, Boost version?

Reply:
Oh I see. There is a different error, probably a result of gorci leftover

Reply:
Few problems appeared afted merge: one line were duplicated (I've fixed it in my local repository copy).
But now there is a problem with orc_b opcode. There is no such opcode definition in opcodes-rv32b file in this repo version (but there were no merge conflicts with opcodes-rv32b file somehow). But this definition present in such file if i do git clone of MIPT-ILab/mipt-mips. How could that be?

Reply:
There were problems with submodules updates.